pkg/pagerduty/pagerduty.go

```diff
@@ -61,10 +61,41 @@ func (p PagerDuty) MoveToEscalationPolicy(incident Incident, escalationPolicy Es
 		},
 	}
 	err := p.manageIncident(o)
-	if errors.Is(err, FailedToUpdateIncidentError{}) {
-		return pkgerrors.Wrap(err, "could not update the escalation policy")
+	if err != nil {
+		return fmt.Errorf("could not update the escalation policy: %w", err)
 	}
-	return err
+	return nil
+}
+
+// AssignToCurrentUser will assign the incident to the current logged in user
+func (p PagerDuty) AssignToCurrentUser(incident Incident) error {
+	err := p.AssignToUser(incident, p.currentUser)
+	if err != nil {
+		return fmt.Errorf("could not assign to the current user: %w", err)
+	}
+	return nil
+}
+
+// AssignToUser will assign the incident to the curr
+func (p PagerDuty) AssignToUser(incident Incident, user User) error {
+	o := []sdk.ManageIncidentsOptions{
+		{
+			ID: incident.ID,
+			Assignments: []sdk.Assignee{
+				{
+					Assignee: sdk.APIObject{
```

Comment:
This formatting could be a little more compact so we.

Reply:
Yeah, but this will hide some stuff?
I'll try and compact it